tests/Unit/Traits/LdapUniqueIdentifierTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Tests\Unit\Traits;
+
+use Mockery as ldapMocker;
+use NetworkRailBusinessSystems\UserLogin\Tests\TestCase;
+use NetworkRailBusinessSystems\UserLogin\Traits\LdapUniqueIdentifier;
+
+class LdapUniqueIdentifierTest extends TestCase
+{
+    public function testRetrieveUniqueIdentifierFromLdap()
+    {
+        $ldapModelMock = ldapMocker::mock('alias:LdapRecord\Models\ActiveDirectory\User');
```

Comment:
You can use `$this->mock` as part of PHPUnit

Reply:
fixed